### ISSUE TITLE
Is admin interface open source?

### ISSUE THREAD

**Issue by hydro-b** [clyso/chorus#20]
I would like to use the Admin interface (looks useful from the standalone install). Is the Admin UI open source? I cannot find the source code in the repository.

**Comment by arttor** [comment]
yes, web ui is in the same repo https://github.com/clyso/chorus/tree/main/ui right now web client is a bit behind CLI.

also supported in helm chart.
